style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [ ] Kindly use a class selector instead of an element selector as it is stated in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)

Reply:
- [ Optional] As required in the previous review you should use a class selector instead of an element selector , kindly fix this :wink: